examples/pom.xml

```diff
@@ -138,6 +138,18 @@
                         </exclusion>
                     </exclusions>
                 </dependency>
+
+                <dependency>
+                    <groupId>org.apache.ignite</groupId>
+                    <artifactId>ignite-spark</artifactId>
+                    <version>${project.version}</version>
+                </dependency>
+
+                <dependency>
+                    <groupId>org.jboss.netty</groupId>
```

Comment:
Apache spark uses netty for remote communication. Without this Spark was having the following problem (see attached StacTrace): 
A latest version of jboss netty resolved the problem. 
[netty_stack_trace.txt](https://github.com/apache/ignite/files/715795/netty_stack_trace.txt)